WordPress/Classes/ViewRelated/Post/PostSettingsViewController_Internal.h

```diff
@@ -18,6 +18,8 @@ typedef enum {
 
 @property (nonnull, nonatomic, strong) NSArray *sections;
 
+@property (nonnull, nonatomic, strong) NSArray *postMetaSectionRows;
```

Comment:
Would this be better off in the `.m` file? That seems like the default location since this looks to be a private property.

Reply:
Good call - I was following what we did we `sections` as a model but this makes total sense. Done.